Gemfile.lock

```diff
@@ -1,60 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.0)
+    better_errors (2.5.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (10.0.2)
+    byebug (11.0.1)
     coderay (1.1.2)
     commander (4.4.7)
       highline (~> 2.0.0)
     database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubi (1.7.1)
-    event_sourcery (0.22.0)
+    erubi (1.9.0)
+    event_sourcery (0.23.0)
     event_sourcery-postgres (0.8.0)
       event_sourcery (>= 0.14.0)
       pg
       sequel (>= 4.38)
-    highline (2.0.0)
+    highline (2.0.3)
     method_source (0.9.2)
     mustermann (1.0.3)
     pg (0.20.0)
     pry (0.12.2)
       coderay (~> 1.1.0)
       method_source (~> 0.9.0)
-    pry-byebug (3.6.0)
-      byebug (~> 10.0)
+    pry-byebug (3.7.0)
```

Comment:
### [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.7.0

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/pry-byebug)_]





<details>
<summary>Commits</summary>

A change of **63** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/pry-byebug/compare/v3.6.0...v3.7.0).

These are the first 10 commits:
- (c25878d) [Move rubocop to circleCI (#161)](https://github.com/deivid-rodriguez/pry-byebug/commit/c25878dac69e5ae5529f7bfbd54696d60eae2e64)
- (070a403) [Bump rubocop from 0.52.1 to 0.53.0 (#160)](https://github.com/deivid-rodriguez/pry-byebug/commit/070a4034e28b47b82f985e7d6c1cc70a51f4f98c)
- (ff526a8) [Bump simplecov from 0.15.1 to 0.16.0 (#165)](https://github.com/deivid-rodriguez/pry-byebug/commit/ff526a875471e1e9a1907b865bfcb4d012ed293e)
- (281dfee) [Bump simplecov from 0.16.0 to 0.16.1 (#166)](https://github.com/deivid-rodriguez/pry-byebug/commit/281dfee16d7cfa02c9e2cb78739dfd1f936d53d2)
- (d6a18b6) [Bump byebug from 10.0.0 to 10.0.1 (#167)](https://github.com/deivid-rodriguez/pry-byebug/commit/d6a18b678b7799c43098b2c6abb1e1a76d256de3)
- (032570e) [Bump rubocop from 0.53.0 to 0.54.0 (#168)](https://github.com/deivid-rodriguez/pry-byebug/commit/032570e71090c678b4f68cdc8a55203476f5b710)
- (0c320b0) [Bump rake from 12.3.0 to 12.3.1 (#169)](https://github.com/deivid-rodriguez/pry-byebug/commit/0c320b0ec1845fb39a8922983f2dad65656eaf5b)
- (5219fb9) [Bump parser to 2.5.0.5 (#170)](https://github.com/deivid-rodriguez/pry-byebug/commit/5219fb980c6c74ca142a0d4daf1aa3566445a9a9)
- (95d322a) [Bump byebug from 10.0.1 to 10.0.2 (#171)](https://github.com/deivid-rodriguez/pry-byebug/commit/95d322adfe5bac99b7f55d1e8acc3b68db40be08)
- (4690e39) [Bump rubocop from 0.54.0 to 0.55.0 (#172)](https://github.com/deivid-rodriguez/pry-byebug/commit/4690e39c0f67fbba468d095f5c83939b8089a976)


</details>